src/bootstrap/src/core/config/flags.rs

```diff
@@ -456,6 +456,15 @@ Arguments:
         #[arg(long)]
         run: bool,
     },
+    /// Vendor dependencies
+    Vendor {
+        /// Additional `Cargo.toml` to sync and vendor
+        #[arg(long)]
+        sync: Vec<PathBuf>,
```

Comment:
Why does this argument exist? Those extra workspaces wouldn't be used at all unless you modify the source code of bootstrap to make use of them, right? At that point you can also modify build_steps/vendor.rs to vendor your extra workspace.